.github/workflows/self-zizmor.yaml

```
@@ -0,0 +1,19 @@
+name: zizmor GitHub Actions static analysis
+on:
+  push:
+  pull_request:
+
+jobs:
+  zizmor:
+    name: Run zizmor from current branch (self test)
+
+    permissions:
+      actions: read
+      contents: read
+
+      pull-requests: write
+      security-events: write
+
+    uses: grafana/shared-workflows/.github/workflows/reusable-zizmor.yml@5946b80e86f32bb4d208c2483c58345bbeef03d2
+    with:
+      codeql-enabled: false
```

Let's get

- https://github.com/grafana/shared-workflows/pull/946
- https://github.com/grafana/shared-workflows/pull/947

merged, so we use an org-wide default configuration, and then

```suggestion
    uses: grafana/shared-workflows/.github/workflows/reusable-zizmor.yml@<the new sha>
```

to use that, and upload the results where we can.

---

OK we're there 👍 I updated the suggestion with the SH